### ISSUE TITLE
Cart is getting removed from the Abandoned Order list

### ISSUE THREAD

**Issue by galakhyati** [TycheSoftwares/Woocommerce-abandon-cart-lite#113]
When admin has abandoned a cart, receives the email for recovery, recovers the cart from the email, the record of abandoned cart is removed from the Abandoned Order tab. Also, the Recovered Order tab does not have a record of recovering the order. Neither the message on the Recovered order is showing there was a cart which was abandoned.

When checked in the DB, the record for the cart id 130 on woo_aclite was also deleted from the abandoned cart history lite table too. 

Also, the admin received an email for the recovery of the cart. And when checked the order#128, the order has an order note saying that the order is recovered. 

Note: This is the first order placed by the admin for the current date.

**Comment by Chetna1510** [comment]
I have tried to replicate the issue on WOO_CHETNA but I am not able to replicate it. I have abandoned a cart as logged-in user(Admin) and recovered an order from abandoned cart reminder email.  The same cart was displayed under the Recovered Orders tab.

It was the first order for current date.

**Comment by Chetna1510** [comment]
I have tried to replicate the issue but I am unable to replicate it. I have abandoned a cart as Admin user and recovered a cart from abandoned cart reminder email. The same cart was displayed under the Recovered Orders tab.

Screenshot of Order Received page: https://www.screencast.com/t/sCjUuIsddyd
Screenshot of Recovered Order tab: https://www.screencast.com/t/QaRkjYx4L

I am closing this issue. As I am not able to replicate this.
